pkg/proof/proof_test.go

```diff
@@ -194,15 +194,15 @@ func TestNewShareInclusionProof(t *testing.T) {
 		},
 		{
 			name:          "blob shares for first namespace",
-			startingShare: 55,
-			endingShare:   57,
+			startingShare: 56,
+			endingShare:   58,
```

Comment:
[no change needed] do you have any idea why these blob shares got move over by one share? Perhaps something in the tx namespace occupied one extra share?

Reply:
Yeah I was actually curious that this needed to change. I looked through the code and couldn't see why the PFBs would take up more space. One thing I think we should add if we don't have already is regression tests to the square / share construction to make sure that breaking changes are caught